TGM/src/main/resources/config.yml

```diff
@@ -69,6 +69,8 @@ server:
   ip: "your.server.ip"
   # Store link displayed when a player tries to join a specific team.
   store: "https://warz.one/store"
+  # Discord link displayed when a player is banned from the Server.
+  discord: "https://discord.io/WarzoneMC"
```

Comment:
I feel like rather than making it the Discord link, we should make it a bit more generic and have it set as "appeal" so that way servers don't have to necessarily link their Discord server and could instead put their forums or whatever else.